### DISCUSSION TITLE
Delayed automatic uploads

### DISCUSSION THREAD

**Discussion by BalooUriza** [Helium314/SCEE#696] [Ideas]
It would be nice to have SCEE automatically upload but only after some time without input to help out slow phones without sacrificing automatic uploads.

**Comment by mnalis** [comment]
On Sat, Nov 30, 2024 at 09:07:25PM -0800, Paul Johnson wrote:
> It would be nice to have SCEE automatically upload but only after some time without input to help out slow phones without sacrificing automatic uploads. 

using "Only on Wi-Fi" as an "Upload answers automatically" might be a
workaround for some (provided you're on mobile data most of the time,
and only connect to wifi from time to time).

**Comment by Helium314** [comment]
I wouldn't even know where to check whether there is some input...
